package.json

```diff
@@ -41,6 +41,7 @@
     "package": "run-func build/yarn-scripts.ts package && yarn --cwd dist/package install && yarn --cwd dist/package vsce package --out ../../",
     "pretest": "yarn compile && yarn webpack",
     "test": "sbt test && node ./out/tests/runTest.js",
```

Comment:
I would say that either before or after `node ./out/tests/runTest.js` you want to run the svelte tests. So either,
```json
"test": "sbt test && yarn test:svelte && node ./out/tests/runTest.js",
```
or
```json
"test": "sbt test && node ./out/tests/runTest.js && yarn test:svelte",
```
I personally prefer the first option but either way is fine with me also. Once that is updated I think this is good to go.

Reply:
I initially did not have the tests being ran during the projects tests because there were a lot of compilation errors within the svelte tests. This is because of #878 

Although, I found out that there are only compilation errors when `mocha` is trying to compile the files with the errors. The example tests that I have do not have any so I went ahead and implemented your changes to the test scripts.